packages/bootstrap/scss/spreadsheet/_theme.scss

```diff
@@ -157,3 +167,23 @@
     }
 
 }
+
+
+
+
+@include exports("spreadsheet/rtl/theme") {
+
+    .k-rtl .k-spreadsheet,
+    .k-spreadsheet[dir="rtl"] {
```

Comment:
If the Spreadsheet can be a child of an "RTL" element, add this selector:

```
[dir="rtl"] .k-spreadsheet
```

Is this an okay scenario?